addons/stock/views/stock_move_views.xml

```diff
@@ -514,31 +514,6 @@
             </field>
         </record>
 
-        <record id="action_receipt_picking_move" model="ir.actions.act_window">
```

Comment:
I don't think we should remove this view, I know it is currently only used in purchase_stock and you removed the menuitem using it, but I'd keep this view nonetheless